### ISSUE TITLE
Unhandled exception when building certain files, then building process terminates

### ISSUE THREAD

**Issue by CookieGod24** [be5invis/Sarasa-Gothic#249]
I am on Manjaro Linux trying to install this. I have all the dependencies after running `npm install`
```
> sarasa-gothic@0.32.6 install
> node checkenv

External dependency <ttx> is present.
External dependency <otc2otf> is present.
External dependency <otf2otc> is present.
External dependency <otf2ttf> is present.
External dependency <otfccdump> is present.
External dependency <otfccbuild> is present.
External dependency <ttfautohint> is present.

up to date, audited 229 packages in 2s

17 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Then I run `npm run build ttf`, which outputs this:
```
> sarasa-gothic@0.32.6 build
> verda -f verdafile.js "ttf"

♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-Regular.otf","o":".build/latin-Inter/Inter-regular.ttf"} 
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-ExtraLightItalic.otf","o":".build/latin-Inter/Inter-extraligh...
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-LightItalic.otf","o":".build/latin-Inter/Inter-lightitalic.tt...
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-Italic.otf","o":".build/latin-Inter/Inter-italic.ttf"} 
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-ExtraLight.otf","o":".build/latin-Inter/Inter-extralight.ttf"} 
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-Light.otf","o":".build/latin-Inter/Inter-light.ttf"} 
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-SemiBold.otf","o":".build/latin-Inter/Inter-semibold.ttf"} 
♦ "make/quadify/index.js" {"main":"sources/Inter/Inter-BoldItalic.otf","o":".build/latin-Inter/Inter-bolditalic.ttf"} 
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-regular.ttf":
 Error: Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:769:11)
    at Socket._write (node:net:781:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at BufferedWriter.flush (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:39:15)
    at BufferedWriter.push (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:35:47)
    at Object.exports.fontJsonStringifyToStream (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:57:10) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-extralight.ttf":
 Error: Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:769:11)
    at Socket._write (node:net:781:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at BufferedWriter.flush (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:39:15)
    at BufferedWriter.push (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:35:47)
    at Object.exports.fontJsonStringifyToStream (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:57:10) {                                                                                                                   
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)                                                                                                                     
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-extralightitalic.ttf":
 Error: Error: write EPIPE
    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)                                                                                                                     
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-lightitalic.ttf":
 Error: Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:769:11)
    at Socket._write (node:net:781:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at BufferedWriter.flush (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:39:15)
    at BufferedWriter.push (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:35:47)
    at Object.exports.fontJsonStringifyToStream (/home/manji/Sarasa-Gothic-master/make/common/support/json-util.js:57:10) {                                                                                                                   
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)                                                                                                                     
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-semibold.ttf":
 Error: Error: write EPIPE
    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)                                                                                                                     
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-light.ttf":
 Error: Error: write EPIPE
    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-bolditalic.ttf":
 Error: Error: write EPIPE
    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
× Error Unhandled exception when building "Builtin::File::.build/latin-Inter/Inter-italic.ttf":
 Error: Error: write EPIPE
    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
    at ChildProcess.<anonymous> (/home/manji/Sarasa-Gothic-master/node_modules/verda/lib/actions/nodejs/command.js:53:31)
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)
Building process terminated.⏎ 
```
Also, the building process still eats up my CPU and is visible in htop.

I am rather new to building from git, am I missing something? Thanks in advance.

**Comment by CyanoHao** [comment]
It should have been fixed in [otfcc#69](https://github.com/caryll/otfcc/pull/69). Is your otfcc binary built from latest source?

**Comment by CookieGod24** [comment]
Yep, that seems to be the problem. I deleted and built otfcc from source again and it worked.
